src/index.rst

```diff
@@ -577,35 +577,42 @@ files from $CTRL_BPS_PANDA_DIR: ::
 setup_panda.sh sets up the PanDA and Rubin environment. ::
```

Comment:
I find it confusing that this is called setup_panda.sh  and there's a different setup_panda.sh in the panda_env directory.   I would probably rename the setup_panda.sh that sets up PanDA and Rubin stack, but minimally the one in the panda_env directory could be renamed to setup_panda_cern.sh.

Reply:
@wguanicedew What do you think, which one should be changed? If you will change the one in panda_env to setup_panda_cern.sh, then I'll update accordingly.

Reply:
I can upgrade to rename it to setup_panda_cern.sh. However I am worried that someone is using it, it will break some users' jobs. I am thinking to add the setup_panda_cern.sh at first, then we can announce it in the panda and cm channels, then we can remove setup_panda.sh in one week.

Reply:
You are right, very likely it'll break the CM tools. if setup_panda.sh isn't harmful, maybe you can  just leave it there. When you add the setup_panda_cern.sh, I'll update the guidance here, so future users will land on the not-so-confusing script name.

Reply:
If easier, could rename the script that users download and source (e.g., from setup_panda.sh to setup_lsst_panda.sh or something better).

Reply:
I see Wen has already added the new script, waiting for it to go to cvmfs.
I'll change the script name in ctrl_bps_panda to setup_lsst_panda.sh as well.